libs/app-example-states/src/lib/appointment-state.service.ts

```diff
@@ -0,0 +1,92 @@
+import { inject, Injectable } from '@angular/core'
+import { SimpleStore } from '@myorg/store'
+import { Appointment } from '@myorg/app-example-models'
+import { combineLatest, debounceTime, switchMap, tap } from 'rxjs'
+import { AppointmentApiService } from '@myorg/app-example-api-services'
+import { AlertService } from '@myorg/app-example-core'
+
+interface AppointmentState {
+    appointments: Appointment[]
+    selectedAppointment: Appointment | null
+    loading: boolean
+    search: string
+    page: number
+    size: number
+    total: number
+}
+
+const initialState: AppointmentState = {
+    appointments: [
+        {
+            vendorUserId: '23rf2',
+            clientUserId: '3232',
+            date: '23232',
+            description: 'hello',
+            endTimestamp: '2323232',
+            notesForClient: 'sfsd',
+            notesForVendor: 'sjkjjjjjj',
+            createdAt: '2323',
+            startTimestamp: '3432232',
+            groupId: '2323',
+            updatedAt: '232332',
+            id: '1',
+        },
+    ],
+    selectedAppointment: null,
+    loading: false,
+    search: '',
+    page: 1,
+    size: 10,
+    total: 0,
+}
+
+@Injectable()
+export class AppointmentStateService extends SimpleStore<any> {
```

Comment:
```suggestion
export class AppointmentStateService extends SimpleStore<AppointmentState> {
```

No `any` please.

- [ ] Also, the state should be renamed to `AppointmentListState` because it is handling multiple appointments in the state. We should reserve the name `AppointmentState` for dealing with single appointment (if required in future).